hardhat.config.ts

```diff
@@ -87,7 +87,7 @@ const config: HardhatUserConfig = {
         },
       },
       {
-        version: "0.8.20",
+        version: "0.8.19",
```

Comment:
We can continue using 0.8.20, and fix the compiler output to `paris`, setting this:

```
settings: {
   optimizer: { ... },
   evmVersion: "paris",
   outputSelection: { ... },
}
```

We can check the bytecode generated doesn't include `PUSH0` in https://www.evm.codes/playground, pasting the bytecode generated at `artifacts/contracts/Governance/VTreasuryV8.sol/VTreasuryV8.json` after executing `yarn compile`.